llvm/test/CodeGen/ARM/misched-branch-targets.mir

```diff
@@ -0,0 +1,181 @@
+# RUN: llc -o - -run-pass=machine-scheduler -misched=shuffle %s | FileCheck %s
+# RUN: llc -o - -run-pass=postmisched %s | FileCheck %s
+
+--- |
+  target datalayout = "e-m:e-p:32:32-Fi8-i64:64-v128:64:128-a:0:32-n32-S64"
```

Comment:
The inline LLVM IR module can probably be simplified by dropping non-essential attributes, as suggested in [TestingGuide](https://llvm.org/docs/TestingGuide.html). [MIR-specific](https://llvm.org/docs/MIRLangRef.html#simplifying-mir-files) documentation also suggests that lots of autogenerated YAML attributes are likely to be safe to remove, but many are dropped already in this test.

I guess the machine functions themselves can be made simpler by dropping `CFI_INSTRUCTION`s and so on, but I'm not sure.

Reply:
Thanks for the suggestions. I've removed a good chunk of unnecessary info.